ribs.py

```diff
@@ -105,12 +105,30 @@ def key_pressed(key):
            (keycode not in last_frame_held_buttons)
 
 
+#
+# Simple sprite drawing
+#
+
+def draw_centered(img, x, y):
+    w, h = img.get_size()
+    window = pg.display.get_surface()
+    window.blit(img, (int(x - w / 2.0), int(y - h / 2.0)))
+
+
+def clear_screen(color):
+    window = pg.display.get_surface()
+    top_left = (0, 0)
+    bottom_right = pg.display.get_surface().get_size()
+    pg.draw.rect(window, color, (top_left, bottom_right))
```

Comment:
Maybe add a comment like:
`# Draw a large rectangle to cover/clear the screen`

Reply:
Not sure how long this code will be there. And how high should the comment to code factor be? Is it really that much easier for them to read heavily commented code?I think there's a sweetspot, and I'm kinda unsure where it is.

I honestly want to remove half of these comments, since they're completely pointless. Given the point you made about not teaching them bad habits, I'd say over commenting your code is one of them.

// Copy of comment to Frans, but relevant here too.

Reply:
That's a good point. I'd say that in this case, the name of the function indicates what it does well enough to not need a comment.

In general though, since these are people who are new to both gamedev and programming, we may want a few more comments than a normal project since what is usually obvious will not be

Reply:
I agree with the point that there should be more comments, but you avoid the more interesting question of what should be in the comments? I don't personally agree with the comment supplied here, but that doesn't mean other people find value there.

What comments are best to write? And when does it just become noise?